sdk/identity/identity/src/msal/nodeFlows/msalClientCertificate.ts

```diff
@@ -170,9 +172,9 @@ export class MsalClientCertificate extends MsalNode {
       // Even though we're providing the same default in memory persistence cache that we use for DeviceCodeCredential,
       // The Client Credential flow does not return the account information from the authentication service,
       // so each time getToken gets called, we will have to acquire a new token through the service.
-      return this.handleResult(scopes, this.clientId, result || undefined);
+      return handleMsalResult(scopes, result || undefined);
```

Comment:
Do you think we should add the "clientId" arg back since that was related to the credential? Was the client id used in the result object available to the user OR the logging?

Reply:
clientId was only needed because `handleResult` was calling `msalToPublic` which is no longer the case. It wasn't used for logging or anything